### DISCUSSION TITLE
Error: Unexpected error occurred: Not Found

### DISCUSSION THREAD

**Discussion by jgttech** [bobheadxi/deployments#80] [Help]
When I attempt to run my deployment I get an error stating I am missing the `deployment_id`. I added this key to the `start` step using the `github.event.deployment.id` but this is `null` and does not actually have anything. So, I tried using something else and ended up landing on using the PR SHA. However, when I use that I get the error message that is this title. I tried several other id's, such as the `github.event.pull_request.base.repo.id` and other `id` values in the Github Context. Everything I try is this same error over and over. If I attempt to remove the `deployment_id` it gets mad at me and states that I need to have one. Is anyone else experiencing something similar to this?

**Comment by bobheadxi** [comment]
Do you mind sharing your full workflow YAML? A full workflow YAML would be helpful, since it is hard to tell what you are trying to do :)
